packages/mui/src/hooks/useDataGrid/index.ts

```diff
@@ -263,6 +272,32 @@ export function useDataGrid<
     };
   };
 
+  const { mutate, isLoading: formLoading } = useUpdate();
```

Comment:
We can also use generics here just like I've mentioned above for the `updateMutationOptions` 🤔 

```suggestion
  const { mutate, isLoading: formLoading } = useUpdate<TData, TError, TData>();
```